rebasehelper/plugins/build_log_hooks/files.py

```diff
@@ -323,15 +327,16 @@ def _correct_deleted_files(cls, rebase_spec_file, files):
         return result
 
     @classmethod
-    def run(cls, spec_file, rebase_spec_file, results_dir, **kwargs):
+    def run(cls, spec_file: SpecFile, rebase_spec_file: SpecFile, results_dir: str,
+            **kwargs: Dict[str, Any]) -> Tuple[Dict[str, Union[RemovedFiles, AddedFiles]], bool]:
```

Comment:
I don't think `**kwargs` is supposed to be `Dict`. It should be `**kwargs: Any` instead.